src/migrations/20220601191441-fix-reimburse-amounts.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint @typescript-eslint/ban-ts-comment: "off" */
+// @ts-nocheck
+
+module.exports = {
+  async up(db) {
+    const collection = db.collection("medici_transactions")
+
+    // Fetch all reimbursement txns
+    let reimbursementTxnsCursor
+    try {
+      reimbursementTxnsCursor = collection.find({ type: "fee_reimbursement" })
+    } catch (error) {
+      console.log({ result: error }, "Couldn't find reimbursement txns")
+    }
+
+    const txns = []
+    for await (const txn of reimbursementTxnsCursor) {
+      txns.push(txn)
+    }
+
+    // Filter all hashes for reimbursement txns
+    const hashesSet = new Set(txns.map((txn) => txn.hash))
+    const hashes = Array.from(hashesSet.values())
+
+    // Update txns in db
+    for (const hash of hashes) {
+      const btcTxn = txns.find(
+        (txn) => txn.hash === hash && txn.currency === "BTC" && txn.debit === 0,
+      )
+      if (btcTxn === undefined) {
+        console.log(`Couldn't find usable reimbursements for ${hash}`)
+        continue
+      }
+
+      const update = {
+        satsAmount: btcTxn.credit,
+        centsAmount: Math.floor(btcTxn.usd * 100),
+        displayAmount: Math.floor(btcTxn.usd * 100),
```

Comment:
are we sure this is in cents?

Reply:
Yup, it's added as the base unit in the `PaymentFlow` object and in the db (below sample from staging). The idea is to not have floats in the system eventually I believe.

![image](https://user-images.githubusercontent.com/17693119/171721343-c1a9900f-6f2d-423c-bfb5-1e96fe882d94.png)